### DISCUSSION TITLE
Bot saying I'm not apart of the Student Developer Pack when I am

### DISCUSSION THREAD

**Discussion by hunter-0404** [community/community#17793] [GitHub Education]
The bot says that it's "not seeing a valid student developer pack approval", yet I am approved and I can confirm this when I go to the site
![image](https://user-images.githubusercontent.com/26722651/166567696-834f8bd2-b115-48f6-a182-f4dc7420b939.png)
How can I fix this?

**Comment by juanpflores** [comment]
Hi @Nanofaux 
Thanks for opening this discussion. We have moved ahead and approved your pull request ⭐ Congratulations on graduating
